includes/class-wc-gateway-stripe.php

```diff
@@ -350,7 +350,18 @@ public function admin_scripts() {
 
 		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
-		wp_enqueue_script( 'woocommerce_stripe_admin', plugins_url( 'assets/js/stripe-admin' . $suffix . '.js', WC_STRIPE_MAIN_FILE ), array(), WC_STRIPE_VERSION, true );
+		wp_register_script( 'woocommerce_stripe_admin', plugins_url( 'assets/js/stripe-admin' . $suffix . '.js', WC_STRIPE_MAIN_FILE ), array(), WC_STRIPE_VERSION, true );
+
+		$params = array(
+			'time'             => time(),
+			'i18n_out_of_sync' => wp_kses(
+				__( '<strong>Warning:</strong> server time is out of sync with the time on your device. Webhook verification depends on the correct time, so please <strong>check your server time</strong> before setting a webhook secret.', 'woocommerce-gateway-stripe' ),
```

Comment:
I'm not sure your average merchant will appreciate the importance of webhook verification. Maybe reword something like this? (and this will also make it make more sense if the time gets out of sync after they've set the secret)

Warning: your site's time appears to be incorrect. Payment webhooks depend on the site time being correct and an incorrect time can result in lost sales. Please contact your hosting provider's support to <strong>correct your site's time</strong>.

Reply:
I'm in favor of some aspects of that change, but I think we _may_ want to avoid assuming:
- correct device time
- that sales could be affected (only true for certain payment methods)
- that they would rather get the time corrected than not have a webhook secret (but then again, maybe we should always encourage signature verification)
- they cannot change the server time themselves (this one's not a big deal)

Taking another swing at it, but feel free to revise further:

> **Warning:** server time is out of sync with the time on your device, and may be incorrect. Verifying webhooks _with signing secret_ depends on the correct time on the host – and webhooks are required for payment processing in some cases – so please <strong>check your server time</strong> before setting a webhook secret (and contact your hosting provider to correct the site's time if necessary).

Reply:
Good idea. A little more wordsmithing:

> Warning: your site's time does not match the time on your browser, and may be incorrect. Some payment methods depend on webhook verification and verifying webhooks with a signing secret depends on your site's time being correct, so please check your site's time before setting a webhook secret. You may need to contact your site's hosting provider to correct the site's time if necessary.